plugins/nlp/nlp.cpp

```diff
@@ -0,0 +1,99 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2021 HPCC Systems®.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+#include "platform.h"
+
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>
+#include <assert.h>
+
+#include "eclrtl.hpp"
+#include "jutil.hpp"
+#include "nlp.hpp"
+
+#define NLP_VERSION "nlp plugin 1.0.0"
+
+ECL_NLP_API bool getECLPluginDefinition(ECLPluginDefinitionBlock *pb)
+{
+    /*  Warning:    This function may be called without the plugin being loaded fully.
```

Comment:
@richardkchapman this comment comes from the example plugin, but is it true?

Reply:
I suspect it used to be true, on Windows only, a long time ago, when we loaded plugins in the compiler using the DONT_RESOLVE_DLL_REFERENCES flag to get access to the ecl plugin definition block only. It's not true any longer AFAIK

Reply:
Why marked as resolved?  Better to delete the out of date comment.